src/finn/custom_op/fpgadataflow/matrixvectoractivation.py

```diff
@@ -67,7 +78,7 @@ def get_nodeattr_types(self):
             "SIMD": ("i", True, 0),
             "MW": ("i", True, 0),
             "MH": ("i", True, 0),
-            "resType": ("s", False, "lut", {"auto", "lut", "dsp"}),
+            "resType": ("s", False, "dsp", {"auto", "lut", "dsp"}),
```

Comment:
I am not sure why the default is changed for this attribute? This could also be an HLS-specific node attribute.